packages/contracts/src/utils/order_factory_from_scenario.ts

```diff
@@ -0,0 +1,277 @@
+import { assetProxyUtils, generatePseudoRandomSalt } from '@0xproject/order-utils';
+import { Order } from '@0xproject/types';
+import { BigNumber, errorUtils } from '@0xproject/utils';
+
+import { DummyERC721TokenContract } from '../generated_contract_wrappers/dummy_e_r_c721_token';
+
+import { constants } from './constants';
+import {
+    AssetDataScenario,
+    ERC721TokenIdsByOwner,
+    ExpirationTimeSecondsScenario,
+    FeeRecipientAddressScenario,
+    OrderAssetAmountScenario,
+    OrderScenario,
+    TakerScenario,
+} from './types';
+
+const TEN_UNITS_EIGHTEEN_DECIMALS = new BigNumber(10000000000000000000);
```

Comment:
Can we use numeric separators or exponentiation here for readability?
https://blog.mariusschulz.com/2018/02/16/typescript-2-7-numeric-separators